themes/scss/map/attributions.scss

```diff
@@ -40,6 +40,9 @@ $size: 20px;
   vertical-align: middle;
   z-index: 1;
 }
+.CDB-Attribution-button.disabled {
```

Comment:
not really, those cases are not covered in the .CDB-Button (the icon is a .CDB-IconFont, not a svg, no white button, ...)

<img width="470" alt="screen shot 2016-11-28 at 15 07 23" src="https://cloud.githubusercontent.com/assets/36676/20671135/779f5592-b57c-11e6-9e94-e93c91fe1632.png">